src/config/env/production.js

```diff
@@ -0,0 +1,15 @@
+export default {
+    env: 'production',
+    db: 'mongodb://localhost/db',
+    port: process.env.SH_BACKEND_PORT || 3000,
+    host: '0.0.0.0',
+    seedDB: true,
+    staticPath: process.env.SH_PATH_FRONTENT_DIST,
```

Comment:
I thought about this, but decided to keep all env variables in same place, as it will be easily monitor external variables the program is depend on.